### PR TITLE
feature: team's bank account view

### PR DESCRIPTION
**Pull request by farrellh1**
# Description

Creating Frontend for Banking functionality in cnc for details of user stories please check the issue below

Fixes # (#124 )

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

* **Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: app/src/components/modals/TransferFromBankModal.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        This will transfer {{ amount }} {{ NETWORK.currencySymbol }} from the team bank contract to
+        this address {{ to }}.
+      </h3>
+
+      <label class="input input-bordered flex items-center gap-2 input-md mt-2">
+        <p>To</p>
+        |
+        <input type="text" class="grow" v-model="to" />
+      </label>
+
```
Comment (dasarathg68) [inline app/src/components/modals/TransferFromBankModal.vue:22]: I think u can display the bank contract's balance in this modal.

File: app/src/services/bankService.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  teamApi: TeamAPI
+  createBankContract(id: string): Promise<string>
+  deposit(bankAddress: string, amount: string): Promise<any>
+  pushTip(bankAddress: string, addresses: string[], amount: number): Promise<any>
+  sendTip(bankAddress: string, addresses: string[], amount: number): Promise<any>
+}
+
+export class BankService implements IBankService {
+  web3Library: IWeb3Library
+  teamApi: TeamAPI
```
Comment (dasarathg68) [inline app/src/services/bankService.ts:17]: why is this needed here?

**Review by dasarathg68 — COMMENTED** [review]
Looks good!

File: app/src/services/bankService.ts (hunk 2 above)
Comment (farrellh1) [reply app/src/services/bankService.ts:17]: - web3Library: because i need it to do some of web3 stuff in the service
- teamApi: i use it because i need to update the bankAddress to deployed bank contract address using update endpoint

File: app/src/components/modals/TransferFromBankModal.vue (hunk 1 above)
Comment (farrellh1) [reply app/src/components/modals/TransferFromBankModal.vue:22]: is this alright?
image

**Review by hermannleboss — APPROVED** [review]
